examples/http-timeout/http_timeout.bal

```diff
@@ -1,46 +1,28 @@
 import ballerina/http;
-import ballerina/log;
 import ballerina/lang.runtime;
 
-http:Client backendClientEP = check new ("http://localhost:8080", {
-        // Timeout configuration.
-        timeout: 10
-
-    });
+final http:Client backendClientEP = check new ("http://localhost:8080", {
+    // Timeout configuration.
+    timeout: 10
+});
 
 // Create an HTTP service bound to the listener endpoint.
 service /timeout on new http:Listener(9090) {
 
-    resource function get .(http:Caller caller, http:Request request) {
-        http:Response|error backendResponse =
-            backendClientEP->forward("/hello", request);
+    resource function get .() returns http:InternalServerError|string|error? {
+        string|error backendResponse =
+                        backendClientEP->get("/hello");
 
         // If `backendResponse` is an `http:Response`, it is sent back to the
```

Comment:
We may need to change the doc hear we take the payload instead of the `http:Response`

Reply:
Fixed it.